nix/modules/scripts.nix

```diff
@@ -35,8 +35,15 @@
         set -xeuo pipefail
         trap "cd $PWD" EXIT
 
-        export VERSIONS_DIR="./versions/''${1}"
-        export DEFAULT_VERSIONS_DIR="$(nix flake metadata --no-write-lock-file --json | jq --raw-output '.locks.nodes.versions.locked.path')"
+        # Check the version of the format, otherwise processing the output might fail silently, like using jq again below
+        FLAKES_LOCK_VERSION=$(nix flake metadata --no-write-lock-file --json | jq --raw-output '.locks.version')
+        if [[ "$FLAKES_LOCK_VERSION" != "7" ]]; then
+          echo "Flakes lock version has changed, refusing to update"
+          exit 1
+        fi
+
+        export VERSIONS_DIR="versions/''${1}"
+        export DEFAULT_VERSIONS_DIR="$(nix flake metadata --no-write-lock-file --json | jq --raw-output '.locks.nodes.versions.locked.dir')"
```

Comment:
There is no field named `path` at this lockfile version so this query was always returning `"null"`.

Reply:
this was the first update run after the change: https://github.com/holochain/holochain/actions/runs/7988887445/job/21814397783